src/pybind/mgr/dashboard/run-backend-api-tests.sh

```diff
@@ -126,7 +126,7 @@ run_teuthology_tests() {
     export COVERAGE_FILE=.coverage.mgr.dashboard
     find . -iname "*${COVERAGE_FILE}*" -type f -delete
 
-    python ../qa/tasks/vstart_runner.py --ignore-missing-binaries $OPTIONS $(echo $TEST_CASES)
+    python ../qa/tasks/vstart_runner.py --ignore-missing-binaries --no-verbose $OPTIONS $(echo $TEST_CASES)
```

Comment:
Would it be better to make ```--no-verbose``` setting to added in this script so we can have the verbosity when running locally?  Sth like:
VERBOSE=true run-backend-api-tests.sh

Reply:
Yup, though about that, but didn't want to add more new input arguments/variables, for the sake of simplicity. If we find we need more verbosity, we could easily add this back, but I'd still go for selectively increasing the verbosity, rather than adding 540.000 extra lines of logs.